Cargo.toml

```diff
@@ -10,4 +10,6 @@ license = "Apache-2.0 OR BSD-3-Clause"
 edition = "2018"
```

Comment:
Can you add some more details to this commit. It adds an important and significant feature to this crate, and it would be nice to be able to tell from the commit description why it was introduced, and other details that are not apparent when reading the code.